installing/installing_with_agent_based_installer/installing-with-agent.adoc

```diff
@@ -0,0 +1,18 @@
+:_content-type: ASSEMBLY
+[id="installing-with-agent"]
+= Installing with Agent
+include::_attributes/common-attributes.adoc[]
+:context: installing-with-agent
+
+toc::[]
+
+== Agent-based installer
+Agent-based installer is a combination of the Assisted Installer (AI) technology and the installer-provisioned infrastructure(IPI). It is a subcommand that resides in the {product-title} installer and also utilizes the {product-title} installer for the input asset management.
```

Comment:
Agent-based installer for OpenShift

Reply:
As discussed above and note that the Agent-based installer content is going to be part of Installing content so  Installing content is all about diff ways/platforms/UPI/IPI etc  to install an OpenShift cluster. ref (https://docs.openshift.com/container-platform/4.11/installing/index.html)